src/wolff_producer.erl

```diff
@@ -720,10 +772,9 @@ handle_overflow(#{replayq := Q,
   ok = replayq:ack(NewQ, QAckRef),
   {FlatBatch, Calls} = get_flat_batch(Items, [], []),
   [_ | _] = FlatBatch, %% assert
-  FlatBatchSize = length(FlatBatch),
-  wolff_metrics:dropped_queue_full_inc(Config, FlatBatchSize),
-  wolff_metrics:dropped_inc(Config, FlatBatchSize),
-  wolff_metrics:queuing_change(Config, -FlatBatchSize),
+  wolff_metrics:dropped_queue_full_inc(Config, 1),
+  wolff_metrics:dropped_inc(Config, 1),
```

Comment:
Maybe this should be `math:ceil(Overflow / BytesLimit)`, so this is the number of _batches_ that _would be_ sent to Kafka?

Reply:
:heavy_check_mark: 
https://github.com/kafka4beam/wolff/pull/41/commits/baf7c688fd9619d25225bd35413642f192d669e5